man/git-alias.md

```diff
@@ -3,9 +3,9 @@ git-alias(1) -- Define, search and show aliases
 
 ## SYNOPSIS
 
-`git-alias`
-`git-alias` &lt;search-pattern&gt;
-`git-alias` &lt;alias-name&gt; &lt;command&gt;
+`git-alias`  
```

Comment:
Is this trailing whitespace necessary?

Reply:
Yes, that's is the markdown syntax for a line break.

Reply:
Would it be better to use multiple newlines?

Reply:
No, that would create an empty line between the different command variants. Which is not the style used by other commands in git-extras (git bulk for example) or in git-core (git stash for example). I'd think it was good if the same style was used in git-extras as in git-core etc.